qcodes/instrument/sims/lakeshore_model335.yaml

```
@@ -0,0 +1,102 @@
+spec: "1.0"
+devices:
+  device 1:
+    eom:
+      GPIB INSTR:
+        q: "\r\n"
+        r: "\r\n"
+    error: ERROR
+    dialogues:
+      - q: "*IDN?"
+        r: "QCoDeS, m0d3l, 336, 0.0.01"
```

Does the 335 really return this on idn?
```suggestion
        r: "QCoDeS, m0d3l, 336, 0.0.01"
```